src/blocks/Companies/Companies.tsx

```diff
@@ -10,14 +10,19 @@ import './Companies.scss';
 
 const b = block('companies-block');
 
-export const CompaniesBlock = ({title, images, animated}: CompaniesBlockProps) => {
+export const CompaniesBlock = ({title, description, images, animated}: CompaniesBlockProps) => {
     const theme = useTheme();
     const themedImages = getThemedValue(images, theme) || {};
 
     return (
         <AnimateBlock className={b()} offset={150} animate={animated}>
             <div className={b('content')}>
                 <h2 className={b('title')}>{title}</h2>
+                {description && (
+                    <div className={b('description')}>
+                        <YFMWrapper content={description} modifiers={{constructor: true}} />
+                    </div>
+                )}
```

Comment:
let's change it to component title <Title> which accepts for entry title and subtitle (descriptin) and remove unnecessary styles